openstack/dns/v2/zones/results.go

```diff
@@ -90,6 +93,7 @@ func (r *Zone) UnmarshalJSON(b []byte) error {
 		CreatedAt     gophercloud.JSONRFC3339MilliNoZ `json:"created_at"`
 		UpdatedAt     gophercloud.JSONRFC3339MilliNoZ `json:"updated_at"`
 		TransferredAt gophercloud.JSONRFC3339MilliNoZ `json:"transferred_at"`
+		Serial        interface{}                     `json:"serial"`
```

Comment:
Without this, I was getting scientific notations as described in #209. I saw this was done for `compute/v2/flavors` for `Swap`, so implemented it here.

Is this the best way to go? Or should this wait until a solution in #209 is implemented?